vertx-web/src/main/java/io/vertx/ext/web/impl/RouteState.java

```diff
@@ -35,6 +41,34 @@
  */
 final class RouteState {
 
+  private static final Logger LOG = LoggerFactory.getLogger(RouteState.class);
+
+
+  enum Priority {
+    PLATFORM,
+    SECURITY_POLICY,
```

Comment:
According to https://github.com/vert-x3/vertx-web/issues/2095 and CORS spec we should invert these two.

1. `SECURITY_POLICY`
2. `PLATFORM`

This means we need to review this change as the currently targetted handlers may have dependencies between the 2 which may require an extra priority value in between